src/Sitewards/DBCompare/Question/FileQuestion.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Sitewards\DBCompare\Question;
+
+use Sitewards\DBCompare\Validator\FilePath;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class FileQuestion
+{
+    /** @var QuestionHelper */
+    private $oQuestionHelper;
+    /** @var FilePath */
+    private $oValidator;
+
+    /**
+     * @param QuestionHelper $oQuestionHelper
+     * @param FilePath $oValidator
+     */
+    public function __construct(
+        QuestionHelper $oQuestionHelper,
+        FilePath $oValidator
+    ) {
+        $this->oQuestionHelper = $oQuestionHelper;
+        $this->oValidator      = $oValidator;
+    }
+
+    /**
+     * @param InputInterface $oInput
+     * @param OutputInterface $oOutput
+     * @param string $sQuestion
+     * @return string
+     * @throws \Symfony\Component\Console\Exception\RuntimeException
+     */
+    public function getFilePath(
+        InputInterface $oInput,
+        OutputInterface $oOutput,
+        $sQuestion = ''
+    )
+    {
+        $oFilePathQuestion = new Question($sQuestion);
```

Comment:
Shouldn't we create this object when we call this method? this way we can mock the object to achieve certain behavior during tests...